developer_manual/general/devenv.rst

```diff
@@ -13,7 +13,7 @@ Basic Tools
 
 To be able to develop with ownCloud and also run unit tests, you will need to install `Node.js <https://nodejs.org>`_.
 
-Other required tools will be automatically installed by composer.
+Other required tools, `such as the required PHP modules`_, will be automatically installed by composer.
```

Comment:
link should go to

https://doc.owncloud.org/server/latest/admin_manual/installation/source_installation.html#installing-on-ubuntu-16-04-lts-server

and composer does not install those modules automatically, you have to do it manual